### DISCUSSION TITLE
Update to new template for j5 compatibility

### DISCUSSION THREAD

**Discussion by micker** [templaza/astroid-framework#455] [Q&A] [answered]
Hello i have a site with Astroid Framework 
Framework Version: 2.5.1
Template Version: 1.3.1
how i can update to your new system in order to update to j5
regards

**Comment by sonvnn** [comment, marked as answer]
1. Let's try to backup your website first.
2. Install on dev server download Astroid 3 and upload to your site.
3. Check all system. Backend and Frontend.
4. If system works well. You can start update your livesite.

**Comment by micker** [comment]
ok thanks
